constants.py

```diff
@@ -20,14 +20,18 @@
 
 import json
 import os
+import re
 
 
 def parse_json_from_js(js_file):
     """Extracts JSON object from JS file."""
     text = js_file.read()
-    first_bracket_index = text.find('= {')
-    last_bracket_index = text.rfind('}')
-    json_text = text[first_bracket_index + 2:last_bracket_index + 1]
+    text_without_comments = re.sub(r'//.*\n', r'\n', text)
```

Comment:
Optional: Maybe drop the line altogether? I.e. replace `r'\n[two spaces]//.*\n` by `r'\n'`?

(I can't seem to type two spaces in the github comment, it keeps getting collapsed into 1.)

It might also be worth adding some unit tests for this function (or at least the part of it that does the parsing).

Reply:
Done (I separated the removing comments part into a new function since a test was already present to check if the json parsed is valid. So, I added a new test which just checks if the new functions for removing comments removes all the comments.)